common/common-ui/src/main/res/layout/component_one_line_item.xml

```diff
@@ -97,6 +97,17 @@
         app:primaryText="Item With Switch and Leading Icon"
         app:showSwitch="true" />
 
+    <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
+        android:id="@+id/oneLineListItemSmallImage"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/keyline_2"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/oneLineListSwitchItemWithLeadingIcon"
+        app:leadingIconSize="small"
+        app:primaryText="Item disabled" />
+
```

Comment:
I can't see this new item in the preview. Maybe we can also add one example with large leadingIconSize

Reply:
On it!